src/styles/main.scss

```diff
@@ -1,3 +1,170 @@
+@import "variables";
+@import "stars";
+@import "header";
+
 body {
+  font-family: $family-roboto;
+  color: #060b35;
   margin: 0;
 }
+
+.main {
+  padding: 50px 0;
+  margin: 0 auto;
+}
+
+.catalog {
+  width: 944px;
+  margin: 0 auto;
+  display: grid;
+  gap: 48px;
+  grid-template-columns: repeat(4, 1fr);
+}
+
+@media (max-width: 1023px) and (min-width: 768px) {
+  .catalog {
+    grid-template-columns: repeat(3, 200px);
+  }
+
+}
+
+@media (max-width: 767px) and (min-width: 488px) {
+  .catalog {
+    grid-template-columns: repeat(2, 200px);
+  }
+}
+
+@media (max-width: 487px) {
+  .catalog {
+    grid-template-columns: repeat(1, 200px);
+  }
+}
+
+.card {
+  margin: $margin-center;
+  height: 406px;
+  width: 200px;
+  background-color: #fff;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  box-sizing: border-box;
+  transition-duration: 300ms;
+
+  &__conteiner {
+    width: 166px;
+    margin: $margin-center;
+  }
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin: $margin-center;
+    display: flex;
+    margin-bottom: 40px;
+  }
+
+  &__discription {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin: 0;
+    margin-bottom: 4px;
+  }
+
+  &:hover .card__discription {
+    color: #34568b;
+  }
+
+  &__code {
+    margin: 0;
+    margin-bottom: 16px;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $secondary-color-gray;
+  }
+
+  &_rate {
+    margin-bottom: 24px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__rate-star {
+    background-image: url(../images/star.svg);
+    background-color: $secondary-color-gray;
+  }
+
+  &_rate-reviews {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+  }
+
+  &__price-text {
+    color: $secondary-color-gray;
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &__price-num {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+  }
+
+  &__button {
+    background-color: $buttun-color;
+    border: 1px solid $buttun-color;
+    box-sizing: border-box;
+    border-radius: 5px;
+    height: 40px;
+    width: 166px;
+    text-decoration: none;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    color: #fff;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    text-transform: uppercase;
+
+    &:hover {
+      background-color: #fff;
+      color: $buttun-color;
+    }
+  }
+
+  &:hover {
+    transform: scale(1.2);
+  }
+}
+
+.stars {
```

Comment:
You already set the styles for the `stars` in another file, why are you duplicating it here?